src/Goteo/Controller/Dashboard/ProjectDashboardController.php

```diff
@@ -246,6 +246,9 @@ protected function getEditRedirect($current = null, Request $request = null) {
     }
 
     /**
+
+    /**
+     * @deprecated
      * Project edit (personal)
      * NOTE: Step removed, maintaining the method just in case is coming back some day
      */
```

Comment:
@davidbeig if this action is actually not being used, instead of marking it as deprecated, I'd propose to you to just remove if from the code. This will simplify it.

Thanks to version control, we can restore it whenever we want.

What do you think?